src/index.html

```diff
@@ -11,7 +11,199 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class = "fieldset">
+        <legend>Personal information</legend>
+        <div class = "form-field">
+          <label>
+            Surname:
+            <input
+            type = "text"
+            name = "surname"
+            autocomplete = "off"
+            required
+            >
```

Comment:
please add spaces and check all similar cases.
![image](https://user-images.githubusercontent.com/22824431/224701712-428dcb71-49ea-4acd-9e1d-5b84df7370df.png)
```suggestion
            <input
              type="text"
              name="surname"
              autocomplete="off"
              required
            >
```

Reply:
Yeah, I see, thank you 🙏🏻 
I’ll make some changes